examples/db/datacenters/london.yml

```diff
@@ -1,3 +1,6 @@
+location:
```

Comment:
These positions are not the most important stuff in the datacenters description, they should be defined at the bottom of the file instead. Also please use fun positions for these examples, such as tower bridge and the eiffel tower :smile:

Reply:
Oh yes, I moved them at the bottom of the file and I used better positions :)